example_project/example_project/settings.py

```diff
@@ -53,34 +53,37 @@
 
 WSGI_APPLICATION = 'example_project.wsgi.application'
 
+# Internationalization
+# https://docs.djangoproject.com/en/1.6/topics/i18n/
```

Comment:
Link `404`s. Why do you need an internationalization comment anyway? I don't see any related settings.